app/controllers/rockles_controller.rb

```diff
@@ -4,7 +4,17 @@ class RocklesController < ApplicationController
   def new
     @species = %w(a b c d e)
     @rockle = Rockle.new
-    render 'new'
+    @user_id = current_user.id
+
+  end
+
+  def index
+    @rockles = Rockle.all
+
+    @rockles.each do |rockle|
+      rockle.user = User.find(rockle.user_id)
```

Comment:
Same for this. See below comment. Are you trying to just preload the users? If so you can just do :
@rockles = Rockle.includes(:user)
No need for the each loop.